bump.cljs

```diff
@@ -30,8 +30,8 @@ Options:
   --defaults-files DEFAULTS-FILES     Comma separated files and/or directories with
                                       default spec data. Not used for selecting
                                       which spec names to query/resolve.
-  --skip-remote-query                 Skip image/rpm querying and uuse
-                                      version-default instead (must be specified fully)
+  --only-variables VARIABLE-NAMES     Comma separated list of variables names to resolve.
+                                      Default to all, if omitted or blank.
```

Comment:
Using "Default" here threw me off for a second. Maybe "If omitted or blank then all are resolved".